public/src/css/style.css

```diff
@@ -1535,6 +1535,10 @@ button {
     margin: 0 10px 0 0;
 }
 
+.tool--remove-button {
+    background: #ed5935 !important;
```

Comment:
Annoying if you have to use the `important` here, the fronts tool css is a bit messed up in places.

Reply:
yes, this is not the first use of `important` here :)